.env.development

```diff
@@ -57,3 +57,4 @@ PLOTLY_SERVER_URL='http://localhost:8050'
 AUTH0_SELF_SERVICE_INTEGRATION='true'
 MFE_CONFIG_API_URL='http://localhost:18000/api/mfe_config/v1'
 DEMO_ENTEPRISE_UUID='set a valid enterprise uuid'
+EDX_ACCESS_URL='http://localhost:1991'
```

Comment:
@sameenfatima78 @zamanafzal we don't have this service in local env. What should be the default url here then? 🤔

Reply:
Wondering which service is this pointing to? Does it have a repo?

Reply:
nope, it's private - managed by auth team.

Reply:
I've just added the base url of its own (frontend-app-admin-portal)

Reply:
Setting it to testing env of auth service which is `http://2u-guid-dev.us.auth0.com/`